interpreter/cling/include/cling/Interpreter/Transaction.h

```diff
@@ -450,6 +450,7 @@ namespace cling {
     ///
     void clear() {
       m_DeclQueue.clear();
+      m_DeserializedDeclQueue.clear();
```

Comment:
Yikes! Thanks!

Reply:
But it was not causing any problem since this routine is not used ...